install/kubernetes/cilium/templates/hubble-ui-ingress.yaml

```diff
@@ -22,7 +22,14 @@ spec:
         paths:
           - path: /
             backend:
+              {{- if semverCompare "^1.19-0" $.Capabilities.KubeVersion.Version }}
```

Comment:
It would be nice to have this definied and stored into a variable in _helper.tpl, so that we don't need to duplicate the knowledge, and re-use easier.

Reply:
the above is not a blocker, can be addressed later.

Reply:
@sayboras Good point and it's definitely cleaner. I put the whole backend blocks (and added the pathType for 1.19) in the `_helper.tpl` file.